client/src/components/ChatMessage.tsx

```diff
@@ -85,44 +100,43 @@ export function sanitizePlayerMessage(text: string): string {
     });
 }
 
-export function translateChatMessage(message: ChatMessage): string {
-    if(GAME_MANAGER.state.stateType !== "game"){
-        return "ERROR: chatmessages cant exist outside game"
-    }
+export function translateChatMessage(message: ChatMessage, playerNames: string[]): string {
```

Comment:
that second parameter can definitely be inferred if absent IMO. I think ChatMenu uses function this for chat filtering

Reply:
I wanted to do that.

It needs to check this however
gamemanager.state.statetype === "game"
I dont want to have it set to undefined or something.

Reply:
On line 20, you do 
```tsx
    if (props.playerNames === undefined) {
        props.playerNames = getPlayerNames();
    }
```

Reply:
It would be a good idea to do that here, perhaps